### PR TITLE
Added deleting images

### PR DESCRIPTION
**Pull request by maybee16**
https://trello.com/c/tBgdhnPs/28-educationservice-remove-education-images

### CODE REVIEW

File: src/EducationService.Business/Commands/Education/CreateEducationCommand.cs [hunk 1]
```diff
@@ -23,7 +23,7 @@ public class CreateEducationCommand : ICreateEducationCommand
   {
     private readonly IAccessValidator _accessValidator;
     private readonly IDbUserEducationMapper _mapper;
-    private readonly IEducationRepository _educationRepository;
```
Comment (vorobevamv) [inline src/EducationService.Business/Commands/Education/CreateEducationCommand.cs:26]: _userEducationRepository

File: src/EducationService.Business/Commands/Education/CreateEducationCommand.cs [hunk 2]
```diff
@@ -32,7 +32,7 @@ public class CreateEducationCommand : ICreateEducationCommand
     public CreateEducationCommand(
       IAccessValidator accessValidator,
       IDbUserEducationMapper mapper,
-      IEducationRepository educationRepository,
```
Comment (vorobevamv) [inline src/EducationService.Business/Commands/Education/CreateEducationCommand.cs:35]: _userEducationRepository

File: src/EducationService.Business/Commands/Education/EditEducationCommand.cs [hunk 3]
```diff
@@ -24,15 +24,15 @@ public class EditEducationCommand : IEditEducationCommand
   {
     private readonly IAccessValidator _accessValidator;
     private readonly IHttpContextAccessor _httpContextAccessor;
-    private readonly IEducationRepository _educationRepository;
```
Comment (vorobevamv) [inline src/EducationService.Business/Commands/Education/EditEducationCommand.cs:27]: _userEducationRepository

File: src/EducationService.Business/Commands/Education/EditEducationCommand.cs [hunk 4]
```diff
@@ -24,15 +24,15 @@ public class EditEducationCommand : IEditEducationCommand
...
+    private readonly IUserEducationRepository _educationRepository;
     private readonly IPatchDbUserEducationMapper _mapper;
     private readonly IEditEducationRequestValidator _validator;
     private readonly IResponseCreator _responseCreator;
 
     public EditEducationCommand(
       IAccessValidator accessValidator,
       IHttpContextAccessor httpContextAccessor,
-      IEducationRepository educationRepository,
```
Comment (vorobevamv) [inline src/EducationService.Business/Commands/Education/EditEducationCommand.cs:35]: _userEducationRepository

File: src/EducationService.Business/Commands/Education/RemoveEducationCommand.cs [hunk 5]
```diff
@@ -18,13 +18,13 @@ public class RemoveEducationCommand : IRemoveEducationCommand
   {
     private readonly IAccessValidator _accessValidator;
     private readonly IHttpContextAccessor _httpContextAccessor;
-    private readonly IEducationRepository _educationRepository;
```
Comment (vorobevamv) [inline src/EducationService.Business/Commands/Education/RemoveEducationCommand.cs:21]: _userEducationRepository

File: src/EducationService.Business/Commands/Education/RemoveEducationCommand.cs [hunk 6]
```diff
@@ -18,13 +18,13 @@ public class RemoveEducationCommand : IRemoveEducationCommand
...
+    private readonly IUserEducationRepository _educationRepository;
     private readonly IResponseCreator _responseCreator;
 
     public RemoveEducationCommand(
       IAccessValidator accessValidator,
       IHttpContextAccessor httpContextAccessor,
-      IEducationRepository educationRepository,
```
Comment (vorobevamv) [inline src/EducationService.Business/Commands/Education/RemoveEducationCommand.cs:27]: _userEducationRepository

File: src/EducationService.Business/Commands/Images/CreateImagesCommand.cs [hunk 7]
```diff
@@ -24,7 +24,7 @@ public class CreateImagesCommand : ICreateImagesCommand
     private readonly IImageRepository _repository;
     private readonly IAccessValidator _accessValidator;
     private readonly IHttpContextAccessor _httpContextAccessor;
-    private readonly IEducationRepository _educationRepository;
```
Comment (vorobevamv) [inline src/EducationService.Business/Commands/Images/CreateImagesCommand.cs:27]: _userEducationRepository

File: src/EducationService.Business/Commands/Images/CreateImagesCommand.cs [hunk 8]
```diff
@@ -34,7 +34,7 @@ public class CreateImagesCommand : ICreateImagesCommand
       IImageRepository repository,
       IAccessValidator accessValidator,
       IHttpContextAccessor httpContextAccessor,
-      IEducationRepository educationRepository,
```
Comment (vorobevamv) [inline src/EducationService.Business/Commands/Images/CreateImagesCommand.cs:37]: _userEducationRepository

File: src/EducationService.Business/Commands/Images/RemoveImagesCommand.cs [hunk 9]
```diff
@@ -22,7 +22,7 @@ namespace LT.DigitalOffice.EducationService.Business.Commands.Image
   public class RemoveImagesCommand : IRemoveImagesCommand
   {
     private readonly IImageRepository _imageRepository;
-    private readonly IEducationRepository _educationRepository;
```
Comment (vorobevamv) [inline src/EducationService.Business/Commands/Images/RemoveImagesCommand.cs:25]: _userEducationRepository

File: src/EducationService.Business/Commands/Images/RemoveImagesCommand.cs [hunk 10]
```diff
@@ -32,7 +32,7 @@ public class RemoveImagesCommand : IRemoveImagesCommand
 
     public RemoveImagesCommand(
       IImageRepository imageRepository,
-      IEducationRepository educationRepository,
```
Comment (vorobevamv) [inline src/EducationService.Business/Commands/Images/RemoveImagesCommand.cs:35]: _userEducationRepository

File: src/EducationService.Business/Commands/Education/CreateEducationCommand.cs (hunk 1 above)
Comment (maybee16) [reply src/EducationService.Business/Commands/Education/CreateEducationCommand.cs:26]: Done

**Review by vorobevamv — APPROVED** [review]

**Review by Vasis3038 — APPROVED** [review]

**Pull request merged**